osgp/platform/osgp-core/src/main/java/org/opensmartgridplatform/core/infra/messaging/CoreLogItemRequestMessageSender.java

```diff
@@ -8,42 +8,42 @@
  */
 package org.opensmartgridplatform.core.infra.messaging;
 
-import javax.jms.ObjectMessage;
-import org.opensmartgridplatform.shared.infra.jms.Constants;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
 import org.springframework.jms.core.JmsTemplate;
 
 public class CoreLogItemRequestMessageSender {
   private static final Logger LOGGER =
       LoggerFactory.getLogger(CoreLogItemRequestMessageSender.class);
 
+  @Value("${auditlogging.message.create.json:false}")
+  private boolean createJsonMessage;
+
   @Autowired private JmsTemplate coreLogItemRequestsJmsTemplate;
 
   public void send(final CoreLogItemRequestMessage coreLogItemRequestMessage) {
+    if (coreLogItemRequestMessage == null) {
+      LOGGER.error("coreLogItemRequestMessage is null and will not be send");
+    } else {
```

Comment:
Better ad a return statement, separating argument checking from other logic as suggested with more detailed explanation in comments with `LoggingMessageSender`.